enginetest/queries/insert_queries.go

```diff
@@ -476,6 +476,20 @@ var InsertQueries = []WriteQueryTest{
 		SelectQuery:         "SELECT * FROM mytable WHERE i = 1",
 		ExpectedSelect:      []sql.Row{{int64(1), "hi"}},
 	},
+	/* Disabled, see https://github.com/dolthub/dolt/issues/7638
+	{
+		WriteQuery:          "INSERT INTO mytable (i,s) values (1, 'hi') AS dt(new_i,new_s) ON DUPLICATE KEY UPDATE s=new_s",
+		ExpectedWriteResult: []sql.Row{{types.NewOkResult(2)}},
+		SelectQuery:         "SELECT * FROM mytable WHERE i = 1",
+		ExpectedSelect:      []sql.Row{{int64(1), "hi"}},
+	},
+	{
+		WriteQuery:          "INSERT INTO mytable (i,s) values (1, 'hi') AS dt ON DUPLICATE KEY UPDATE mytable.s=dt.s",
+		ExpectedWriteResult: []sql.Row{{types.NewOkResult(2)}},
+		SelectQuery:         "SELECT * FROM mytable WHERE i = 1",
+		ExpectedSelect:      []sql.Row{{int64(1), "hir"}},
+	},
+	*/
```

Comment:
Are these the new "insert row aliases are not currently supported" error cases? Can we leave them in and assert they return the new error message? That would be a little better than commenting them out. Alternatively, you could add them as skipped tests somewhere – we do track those skipped tests (James has been doing that lately) so at least they are still visible that way.

Reply:
Added a Skip flag to this test type. How does this look?